extensions/amp-story/1.0/amp-story-consent.js

```diff
@@ -114,7 +115,8 @@ const getTemplate = (config, consentId, logoSrc) => ({
                 'on': `tap:${consentId}.reject`,
               }),
               children: [],
-              unlocalizedString: 'Decline',
+              localizedStringId:
```

Comment:
This looks like the wrong id?

Reply:
Oops! Forgot to commit my last edit... Thanks.